inception-project/inception
# #555 - Kb items matching the exact text entered by the user not top ranked

**What's in the PR**
* Fix exact matching for entity linking
* look for labels with and without language (language must be default language of the knowledge base)
* simplify query
- Implemented new SPARQLQueryBuilder to facilitate the building of INCEpTION-oriented queries
- Added unit tests for SPARQLQueryBuilder
- Use SPARQLQueryBuilder to retrieve candidates for entity linking
- Handle case where no langauge is configured in the KB
- Best-effort case-insensitive matching
- Fixed ConceptLinkingService tests
- Removed some commented-out code
- Improved/changed some method signatures

**How to test manually**
* Search for entities in the ConceptFeatureEditor by typing the exact name
* See if the corresponding entity is top ranked

**Automatic testing**

**Documentation**
